crates/fixture/src/lib.rs

```diff
@@ -0,0 +1,53 @@
+use std::fs::{self, File};
+use std::io::prelude::*;
+use std::mem::ManuallyDrop;
+use std::path::PathBuf;
+use std::thread;
+
+use tempfile::TempDir;
+
+pub struct Fixture {
+    dir: ManuallyDrop<TempDir>,
+}
+
+impl Fixture {
```

Comment:
Not a review as much as a praise. This is really really nice to have at our disposal 😍